packages/lesswrong/components/search/CommentsSearchHit.tsx

```diff
@@ -3,14 +3,24 @@ import { Link } from '../../lib/reactRouterWrapper';
 import { Snippet } from 'react-instantsearch-dom';
 import type { Hit } from 'react-instantsearch-core';
 import React from 'react';
+import ChatBubbleOutlineIcon from '@material-ui/icons/ChatBubbleOutline';
 
 const styles = (theme: ThemeType): JssStyles => ({
   root: {
-    marginLeft: theme.spacing.unit,
-    marginBottom: theme.spacing.unit*2
+    padding: 10,
+    paddingTop: 8,
+    paddingBottom: 8,
+    display: 'flex',
+    alignItems: 'center',
+    borderTop: "solid 1px rgba(0,0,0,.1)"
```

Comment:
```suggestion
    borderTop: theme.palette.border.faint
```
(a unit tests complains about this because it doesn't handle dark mode)